src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
> However that is probably less compressible and slower since the property name won't be minified the way a variable is?

Easy to check :)

@timmywil could provide a consult on amd here, i understand why we do this for `toString` or `indexOf` but defining `document` and `documentElement` is that some sort of convention or convenience?